pkg/api/helpers.go

```diff
@@ -84,3 +85,21 @@ func IsPullIfNotPresent(p PullPolicy) bool {
 func pullPoliciesEqual(p1, p2 PullPolicy) bool {
 	return strings.ToLower(string(p1)) == strings.ToLower(string(p2))
 }
+
+func IsStandardResourceName(str string) bool {
+	switch str {
+	case string(ResourceMemory), string(ResourceCPU):
+		return true
+	}
+
+	return false
+}
+
+// Returns fully qualified name of a compute resource.
+func GetQualifiedResourceName(resourceName string) ResourceName {
```

Comment:
Can you move this to util and call it SplitQualifiedName() (return namespace and name), then set the default, then rejoin with JoinQualifiedName() ?